src/store/getters.js

```diff
@@ -118,14 +118,17 @@ export default {
     new BigNumber(transaction.fee || transaction.tx?.fee || 0),
   ),
   getTxDirection: (_, { account: { address } }) => (tx, externalAddress) => {
+    const currentAddress = externalAddress || address;
+
     if (getTxType(tx) === SCHEMA.TX_TYPE.spend) {
-      return tx.recipientId === address
+      return tx.recipientId === currentAddress
```

Comment:
@pstachel and @Liubov-crypto had some concerns about this changes previously.

Reply:
Yes, I did, but you were right that it should look like that